farmdata2_modules/fd2_tabs/resources/CustomTableComponent.js

```diff
@@ -413,13 +413,14 @@ let CustomTableComponent = {
     watch: {
         // Leaving this uncommented code in here until I try this out with something
         // more complex than the simple UI page
-        // columns: {
-        //     // using deep watch to track nested property changes
-        //     handler(newArr) {
-        //         this.updatedVis = newArr
-        //     },
-        //     deep: true
-        // }
+        columns: {
+            // using deep watch to track nested property changes
+            handler(newArr) {
+                console.log("Hey!")
+                this.columns = newArr

```

Comment:
I'm not sure that resetting the prop internally here does what we need to do.  The original code was updating an element of the data, not the prop.  So maybe there is something going on with that?   It would be worth doing a little reading on deep watches and how they are typically used.

Reply:
You may want to put the prop into a data element (as was done with `updatedVis` in the original code).  Then in the watch, update that data element.  Then use that data element in the `v-if`s in the html.

Reply:
Looks like you can also watch for changes just in specific nested properties inside of a prop.  See: https://stackoverflow.com/questions/42133894/vue-js-how-to-properly-watch-for-nested-data

Reply:
Agreed.  Leave it as it is for now and add an issue to make the dropdowns in the seeding report have context sensitive contents based on the type of seeing etc... You'll need to think about it a little in how to write up the issue to describe how the dropdown contents should be computed.  You can also then in a comment on that issue describe the approach to fixing this that we have discussed (i.e. making the prop a computed property rather than the data element).

Reply:
I believe this would fall under #585

Reply:
Similarly, there should be a new issue created that has the edit-clicked event emit the id of the row that is being edited as its payload.

Reply:
That is also mentioned under #585

Reply:
I see that you already have an issue filed that captures both of the above changes.  Thanks!